src/main/java/Amy.java

```diff
@@ -0,0 +1,95 @@
+import java.util.Scanner;
+
+public class Amy {
+    private static final int MAX_TASKS = 100;
+    private static Task[] taskList = new Task[MAX_TASKS];
+    private static int taskNum = 0;
+    public static void doEcho(String line){
+        System.out.println(line + "! I wonder what I can do with this information (●'◡'●)");
+    }
+    public static void getTaskList(){
+        String line = "Let's try to get this done! You got this (๑•̀ㅂ•́)و✧";
+        String noTask = "You don't have anything in your TDL! Take some rest for now (✿◡‿◡)";
+
+        if(taskNum == 0) System.out.println(noTask);
+        else    System.out.println(line);
+
+        for(int i = 0; i<taskNum; i++){
+            System.out.println(String.valueOf(i+1) + '.' + taskList[i]);
+        }
+    }
+    public static void addTaskList(String input){
+        taskList[taskNum++] = new Task(input);
+        System.out.println("I added <" + input + "> to the todo-list! ☆*: .｡. o(≧▽≦)o .｡.:*☆");
+    }
+    public static void doGreeting(){
+        String name = "Amy";
+        String greet = "Hello! I'm " + name + " ✿" + "\nWhat can I do for you? (❁´◡`❁) \n";
+        System.out.println(greet);
+    }
+    public static void exit(){
+        String bye = "Bye. Hope to see you again soon! (*/ω＼*)\n";
+        System.out.println(bye);
+    }
+    public static void filler(){
```

Comment:
would be better if constants can be global and suggest to use all caps names for it